lifecycle-build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+rm -f /home/vcap/app/.cloudfoundry/.placeholder
+rm -f /home/vcap/app/.profile.d/.placeholder
+cd /home/vcap || exit
```

Comment:
I assume `exit` will be exiting with the non-zero exit code that `cd` exited with?  Just making sure, I've actually never seen this `cd <directory-that-might-not-exist> || exit` construction before...